scripts/check_frontend_coverage.py

```diff
@@ -113,6 +113,7 @@
     'copier.directive.ts',
     'correctness-footer.directive.ts',
     'create-activity-button.directive.ts',
+    'create-new-skill.modal.controller.ts',
```

Comment:
Why are we adding a new file to this list? [I think we only expect to remove the file from this ist, right?]

Reply:
Yeah, you're right @DubeySandeep

Reply:
@marianazangrossi The topic dashboard had an inline uibModal, and in order to test this file, I needed to separate the modal out, and hence I created a new file and added that to the list in the scripts/check_frontend_coverage.py. Since the modal is related to skills(i have only worked on topics in this PR), I plan to test that file in the next PR that I'll make and I'll remove this file from this list then(because probably I'll need to make some changes to that modal)
Link for reference: #8924

Reply:
Leaving this for @marianazangrossi to decide. I'm fine with whatever you all decide.

Reply:
I think this is OK, based on what I saw of @marianazangrossi's other PR.